tests/Doctrine/Tests/DBAL/Driver/PDOPgSql/DriverTest.php

```diff
@@ -104,16 +106,18 @@ protected function createDriver()
     }
 
     /**
-     * @throws \PHPUnit_Framework_SkippedTestError
+     * @throws PHPUnit_Framework_SkippedTestError
      */
     private function skipWhenNotUsingPhp56AndPdoPgsql()
     {
         if (! defined('PDO::PGSQL_ATTR_DISABLE_PREPARES')) {
             $this->markTestSkipped('Test requires PHP 5.6+');
         }
 
-        if (! (isset($GLOBALS['db_type']) && $GLOBALS['db_type'] === 'pdo_pgsql')) {
-            $this->markTestSkipped('Test enabled only when using pdo_pgsql specific phpunit.xml');
+        if (isset($GLOBALS['db_type']) && $GLOBALS['db_type'] === 'pdo_pgsql') {
```

Comment:
I wish I never saw this. :see_no_evil:

Reply:
Phew… I thought it was a bug in the CS fixer which just removed the `!`. But it added an early return.

As for the `$GLOBALS`… yeah.